react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,19 @@
+import "./App.css";
+import Todo from "./components/Todo";
+import TodoList from "./components/TodoLists";
+import TodoForm from "./components/TodoForm";
+
+function App() {
+  return (
+    <div className="lists">
+      <h1> Todo list</h1>
+      <p> You have used {} seconds on this page</p>
```

Comment:
You would need a timer here to count the number of seconds I've been on this page. 
Like this: https://github.com/HackYourFuture-CPH/React/blob/master/week2/lesson-plan.md#counter